pkg/pillar/types/zedcloudmetrics.go

```diff
@@ -12,11 +12,12 @@ type MetricsMap map[string]ZedcloudMetric
 
 // ZedcloudMetric are metrics for one interface
 type ZedcloudMetric struct {
-	FailureCount uint64
-	SuccessCount uint64
-	LastFailure  time.Time
-	LastSuccess  time.Time
-	URLCounters  map[string]UrlcloudMetrics
+	FailureCount  uint64
+	SuccessCount  uint64
+	LastFailure   time.Time
+	LastSuccess   time.Time
+	URLCounters   map[string]UrlcloudMetrics
+	AuthFailCount uint64
```

Comment:
Don't you want separate counters for the 5 different status values you defined?
Even counting the misses is useful. [No need to address that know, can be done in the next sprint as a separate PR.]

Reply:
will change at next sprint then.